package.json

```diff
@@ -13,7 +13,7 @@
     "build": "tsc",
     "gen-docs": "typedoc --readme none --mode modules --excludeNotExported --excludePrivate --exclude **/*_test.ts --out ./docs/api .",
     "test": "npm run build && wct",
-    "checksize": "uglifyjs lit-element.js -mc --toplevel | gzip -9 | wc -c",
+    "checksize": "rollup -c ; rm lit-element.bundled.js",
```

Comment:
don't you need a rollup config?

See https://github.com/Polymer/lit-html/blob/master/rollup.config.js

Reply:
Done.